Samples/JavaKitSampleApp/Package.swift

```diff
@@ -30,7 +30,7 @@ func findJavaHome() -> String {
 }
 let javaHome = findJavaHome()
 
-let javaIncludePath = "\(javaHome)/include"
+let javaIncludePath = ProcessInfo.processInfo.environment["JAVA_INCLUDE_PATH"] ?? "\(javaHome)/include"
```

Comment:
Worth calling out in the readme? What do the paths look like for android?

Reply:
Probably not standard but my question was more about "why do we need this" / "why are the paths different and how do they look like" and just adding a note to the README about how to use this

Reply:
We need to pick up the host JVM when building Java2Swift and JavaCompilerPlugin, vs the Android JVM when building JavaKitExample. `JAVA_HOME` and `JAVA_INCLUDE_PATH` serve to disambiguate between the two.

Reply:
Sounds good, can we plop such a note in to the readme somewhere along with this pr please?